py4cast/plots.py

```diff
@@ -390,15 +391,30 @@ def plot_map(
             ]
             # TODO : don't create all figs at once to avoid matplotlib warning
             tensorboard = obj.logger.experiment
+            mlflow_logger_idx = [
+                i for i, l in enumerate(obj.loggers) if isinstance(l, MLFlowLogger)
+            ]
+            mlflow_logger = (
+                obj.loggers[mlflow_logger_idx[0]] if mlflow_logger_idx else None
+            )
```

Comment:
```suggestion
            mlflow_logger = next([o for o in obj.loggers if isinstance(o, MLFlowLogger)], None)
```
This code is duplicated many times, a property in the lightning module might do the trick ?